scripts/missing_tables/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/bmatcuk/doublestar/v4"
+)
+
+func parseDocsTables() map[string]bool {
+	tablesMap := make(map[string]bool)
+	tablesReadmes, err := doublestar.Glob(os.DirFS("../../"), "plugins/source/**/docs/tables/README.md", doublestar.WithFailOnPatternNotExist(), doublestar.WithFilesOnly())
+	if err != nil {
+		panic(err)
+	}
+
+	for _, readme := range tablesReadmes {
+		content, err := ioutil.ReadFile("../../" + readme)
+		if err != nil {
+			panic(err)
+		}
+		lines := strings.Split(string(content), "\n")
+		for _, line := range lines {
+			pos1 := strings.Index(line, "[")
+			if pos1 == -1 {
+				continue
+			}
+			pos2 := strings.Index(line, "]")
+			table := line[pos1+1 : pos2]
+			tablesMap[table] = true
+		}
+	}
+
+	return tablesMap
+}
+
+func parseCodeTables() map[string]string {
+	tablesMap := make(map[string]string)
+	tableFiles, err := doublestar.Glob(os.DirFS("../../"), "plugins/source/**/resources/services/**/*.go", doublestar.WithFailOnPatternNotExist(), doublestar.WithFilesOnly())
+	if err != nil {
+		panic(err)
+	}
+	for _, tableFile := range tableFiles {
+		if strings.HasSuffix(tableFile, "_test.go") {
+			continue
+		}
+		content, err := ioutil.ReadFile("../../" + tableFile)
+		if err != nil {
+			panic(err)
+		}
+		contentString := string(content)
+		if !strings.Contains(contentString, "*schema.Table") {
+			continue
+		}
+
+		tableNameRegex := regexp.MustCompile(`Name\:.*?"(.*?)",`)
```

Comment:
I think maybe this regex could also catch column names by accident?

Reply:
To be absolutely sure we'd probably need to use the AST 😅  But another thing I've done that's a bit easier is to keep track of the curly brackets in a stack, and make sure the `Name` entry is at the correct level (right after the `schema.Table` definition)

Reply:
It doesn't have false positives at the moment. It works since it reports the first match.
I didn't want to AST 😄 as it can get a bit complex/slow.
I think I can make the regular expression a bit more strict (to match first name after table declaration) WDYT?

Reply:
Something like https://github.com/cloudquery/cloudquery/pull/6900/commits/447fdfd77d1507926b2c5a7ec221601f30d9ca74